hw17/html_render.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+"""
+Python class example.
+
+"""
+
+
+class Element(object):
+    IND_LEVEL = "    "
+    name = "html"
+
+    def __init__(self, content="", **kwargs):
+        self.attributes = kwargs
+        self.attr = ""
+        self.children = [content] if content else []
+        for attr_key, attr_val in self.attributes.items():
+            self.attr += "{key}=\"{val}\"".format(key=attr_key, val=attr_val)
```

Comment:
`"{key}=\"{val}\""`
You're missing a space ---
`" {key}=\"{val}\""`
It's subtle, but look at what your HTML looks like in the output. Your tags and attributes are smashing together!
`<pstyle="text-align: center; font-style: oblique;">`